### PR TITLE
Create doc about HTTP

### LINKED ISSUE #14: Create doc about Http

### PR DESCRIPTION
**Pull request by gitkuba**
Fixes #14

### CODE REVIEW

File: http.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+HTTP - short for Hypertext Transfer Protocol; The foundation of data communication for the World Wide Web.
+
+## Classification / acceptance questions
+
+### Apprentice aka _I know it exists_
+- I know what problem is solved by HTTP
+
+### Intermediate aka _I've been looking in the network tab_
+- I know what types of content can be transported via HTTP
+- I know the structure of a HTTP request/response
```
Comment (rogatty) [inline http.md]:
```suggestion
- I know the structure of an HTTP request/response
```

File: http.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- I can make practical use of the network tab to debug a problem
+- I know request types and response status codes
+
+### Advanced aha _?_
```
Comment (rogatty) [inline http.md]: I don't have any good suggestions but if you don't come up with any then just use:
```suggestion
### Advanced
```

**Review by rogatty — APPROVED** [review]

File: http.md (hunk 1 above)
Comment (bognix) [reply http.md]: that is interesting: https://www.quora.com/A-http-or-an-http-which-is-the-correct-grammatical-usage/answer/Rob-Kerr-3

Comment (rogatty) [reply http.md]: Huh, I pronounce it en ejdżtitipi, what would be the other way?

File: http.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# HTTP
+
+HTTP - short for Hypertext Transfer Protocol; The foundation of data communication for the World Wide Web.
+
+## Classification / acceptance questions
```
Comment (kvas-damian) [inline http.md:5]: I think TCP is a prerequisite knowledge here.

File: http.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+### Apprentice aka _I know it exists_
+- I know what problem is solved by HTTP
+
+### Intermediate aka _I've been looking in the network tab_
+- I know what types of content can be transported via HTTP
+- I know the structure of a HTTP request/response
+- I can make practical use of the network tab to debug a problem
```
Comment (kvas-damian) [inline http.md]:
```suggestion
- I can make practical use of the browser network tab to debug a problem
```

File: http.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+- I know request types and response status codes
+
+### Advanced aha _?_
+- I'm familiar with security flaws that come with sending unencrypted data via HTTP and how we solve that problem.
+- I know differences between HTTP versions, their pros and cons and when to use a particular version.
```
Comment (kvas-damian) [inline http.md:18]: I have a feeling it's too broad. Maybe we could name a few features of HTTP2.0?

File: http.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+- I know what problem is solved by HTTP
+
+### Intermediate aka _I've been looking in the network tab_
+- I know what types of content can be transported via HTTP
+- I know the structure of a HTTP request/response
+- I can make practical use of the network tab to debug a problem
+- I know request types and response status codes
+
+### Advanced aha _?_
+- I'm familiar with security flaws that come with sending unencrypted data via HTTP and how we solve that problem.
```
Comment (kvas-damian) [inline http.md:17]: Are you referring to HTTPS? If yes, let's make it clear - without the keyword it's harder to google it.
If not, let's add a point about HTTPS

**Pull request closed without merging**